Update cost_details.md (#20616)
DataDog/documentation: content/en/account_management/plan_and_usage/cost_details.md

## Overview

> Cost Summary and Cost Chargebacks help you understand your estimated month-to-date, projected end-of-month and historical Datadog costs.

Reviewer: ```suggestion
Cost Summary and Cost Chargebacks help you understand your estimated month-to-date, projected end-of-month, and historical Datadog costs.
```

You can break down your costs by sub-organization and by product to:
- Allocate costs according to their source
- Gain insight into how costs are tracking

### Permissions

To view the Cost Summary and Cost Chargebacks data, you must be a Datadog Admin user.

Alternately, roles with Billing Read (`billing_read`) and Usage Read (`usage_read`) [permissions][1] can view the Cost Summary and Cost Chargebacks data.

## Cost summary

Use the cost summary to:
- View estimated month-to-date costs
- View projected end-of-month costs
- View cost trends within the month
- Filter and group costs by product or sub-organization
- View cumulative day-over-day costs

> Projected end-of-month costs are calculated by applying current month projected usage data against your contracted rates. Projections are available around the 12th of the month and are updated daily. Because the costs are a prediction, the amount may differ from your finalized monthly cost.

Reviewer: ```suggestion
Projected end-of-month costs are calculated by applying the current month's projected usage data against your contracted rates. Projections are available around the 12th of the month and are updated daily. Because the costs are a prediction, the amount may differ from your finalized monthly cost.
```

### Cost Summary (parent organization)

> The cost summary functionality changes according to whether you use Datadog as a single organization or a multi-organization. As a multi-organization, you can view estimated and projected costs for the parent organization and each sub-organization. 

Reviewer: ```suggestion
The cost summary functionality changes according to your Datadog usage as a single organization or a multi-organization. As a multi-organization, you can view estimated and projected costs for the parent organization and each sub-organization. 
```

To download the data as a comma separated value file, click **Download as CSV**.

## Cost chargebacks

Use the cost chargebacks to:
- View estimated month-to-date and historical costs for multi-organizations
- Attribute costs to each sub-organization

> Cost chargebacks are dervied by:

Reviewer: ```suggestion
Cost chargebacks are derived by:
```
